config/boards/armsom-w3.wip

```diff
@@ -0,0 +1,29 @@
+# Rockchip RK3588 SoC octa core 8-32GB SoC 2.5GBe PoE eMMC USB3 NvME
+BOARD_NAME="ArmSoM W3"
+BOARDFAMILY="rockchip-rk3588"
+BOARD_MAINTAINER="armsom-team"
+BOOTCONFIG="rock-5b-rk3588_defconfig"
+KERNEL_TARGET="legacy"
+KERNEL_TEST_TARGET="legacy" # in case different then kernel target
+FULL_DESKTOP="yes"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="rockchip/rk3588-armsom-w3.dtb"
+BOOT_SCENARIO="spl-blobs"
+BOOT_SUPPORT_SPI="yes"
+BOOT_SPI_RKSPI_LOADER="yes"
+IMAGE_PARTITION_TABLE="gpt"
+SKIP_BOOTSPLASH="yes" # Skip boot splash patch, conflicts with CONFIG_VT=yes
+BOOTFS_TYPE="ext4"
+
+function post_family_tweaks__rock5b_naming_audios() {
+	display_alert "$BOARD" "Renaming rock5b audios" "info"
```

Comment:
It's better to change rock5b to the real board name.

Reply:
Thank you, let's modify it